handlers/LunchHandler.js

```diff
@@ -121,6 +140,89 @@ export default class LunchHandler extends BaseHandler {
 		);
 	}
 
+	getHotPotMenu() {
+		return new Promise((resolve, reject) => {
+			http.request(optionsPaevapraed, (res) => {
+				let item = '';
+
+				res.setEncoding('utf8');
+				res.on('data', (chunk) => {
+					const $ = cheerio.load(chunk);
+					const chunkHtml = $('#HOTPOT_FOOD').html();
+
+					if (chunkHtml) {
+						item += chunkHtml;
+					}
+				});
+				res.on('end', () => {
+					const itemText = htmlToText.fromString(item);
+					const items = itemText.split('\n');
+					const itemsList = items.length > 2 ? [items[2], items[5], items[8], items[11]] : [items[0]];
```

Comment:
Using items[11] so check for items.length >= 12 ?

Rename items => lines, itemsList => items